modules/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-creating-dns-records-on-infoblox.adoc
+
+:_content-type: PROCEDURE
+[id="nw-control-dns-records-public-dns-zone-infoblox_{context}"]
+= Creating DNS records on an public DNS zone for Infoblox by using Red Hat External DNS Operator
+
+You can create DNS records on a public DNS zone for Infoblox by using the Red Hat External DNS Operator.
+
+.Procedure
+
+. Create a `secret` with Infoblox credentials:
++
+----
+$ oc -n external-dns-operator create secret generic infoblox-credentials --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_USERNAME=${INFOBLOX_USERNAME} --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_PASSWORD=${INFOBLOX_PASSWORD}
+----
+
+. Get the `routes` to check your cluster domain:
++
+[source,terminal]
+----
+$ oc get routes --all-namespaces | grep console
+----
++
+.Example Output
+----
+openshift-console          console             console-openshift-console.apps.test.example.com                       console             https   reencrypt/Redirect     None
+openshift-console          downloads           downloads-openshift-console.apps.test.example.com                     downloads           http    edge/Redirect          None
+----
+
+.From the CLI
```

Comment:
Is this step 3?

Reply:
Technically yes but this procedure is a mixture of both CLI and UI so I tried to give headings like here: https://docs.openshift.com/container-platform/4.7/storage/persistent_storage/persistent-storage-local.html#local-storage-install_persistent-storage-local

Reply:
Ok, so in the example you linked, that is showing the _same_ procedure done with two different methods (two choices). If I am understanding correctly, this is _one_ procedure that has sequential steps in different interfaces (no choice). 

So I think what you have is one procedure as follows:
1. Create secret object
2. Get routes
3. (in the CLI) Create ExternalDNS YAML
4. (in CLI) Use YAML to create the resource
5. (in Infoblox UI) Check records

Is that right? If so, I would remove the CLI and UI headers. Steps 1&2 also seem to be done from the CLI, so you can just run through this flow, and when you get to step 5, you can simply start the sentence with "From the Infoblox UI, ...".

Reply:
Agreed, addressed